recipe/meta.yaml

```diff
@@ -20,7 +20,6 @@ requirements:
   host:
     - python
     - pip
-    - setuptools_scm >=3.3.1
```

Comment:
Why is this being dropped? It is still used [here]( https://github.com/ionelmc/python-lazy-object-proxy/blob/5b36b50b678df414e63e5e819350a03723349931/pyproject.toml#L5 ).

Reply:
Sorry about that.  It was dropped from the `setup.cfg`.  I didn't notice that there is also a `pyproject.toml`.  I'll revert.

Reply:
One of the builds failed.  It could just be a fluke, but I was looking at dependency changes to see if there could be an explanation.  I saw this removed from `setup.cfg` (presumably just clean-up) and thought it might matter.  again, sorry about that.

Reply:
> We might also want to add setuptools (even though pip includes it) because of [this constraint](https://github.com/ionelmc/python-lazy-object-proxy/blob/5b36b50b678df414e63e5e819350a03723349931/pyproject.toml#L3)

Yes, wouldn't hurt.

Reply:
Done.

Reply:
All good. Was just trying to understand the change. Thanks for the context (and the fixes)! 🙏